src/sage/misc/randstate.pyx

```diff
@@ -1005,7 +1005,6 @@ def benchmark_libc():
         125 loops, best of 3: 2.12 ms per loop
     """
     cdef int i
-    cdef randstate rstate = _current_randstate
     for i from 0 <= i < 100000:
```

Comment:
some day we will have to change that to `for I in range(100000):`, but may be in a dedicated PR